.github/workflows/workflow-review.yml

```diff
@@ -1,3 +1,4 @@
+# https://stackoverflow.com/questions/67247752/how-to-use-secret-in-pull-request-review-similar-to-pull-request-target
```

Comment:
is this needed?

Reply:
yeah. We should give credits to whoever came up with the idea. This is not a original idea I came up with.